### ISSUE TITLE
Dataset Release

### ISSUE THREAD

**Issue by Wssduer** [jhmlam/NucleicNet#2]
Dear professor, I am a graduate student at Shandong University in China. I am recently doing research on protein-RNA binding sites and read your paper "A deep learning framework to predict binding preference of RNA constituents on protein surface". This is a very great work and I am very inspired by your team's research and the design of "NucleicNet".
I want to get the datasets and labels of your research to continue my work, could you please send me (shizhuozhang@mail.sdu.edu.cn)? Thank you very much!

**Comment by jhmlam** [comment]
Hi @Wssduer, 

Thank you for your interest! We are preparing to release the code to build the dateset as well as the code to train the model in the coming summer. We are still finalizing the format as the dataset is pretty huge in size (>280GB.) and adjusting to recent changes in PDB (api and obsolete entries). Will surely let you know when we finished.

Best,
Jordy

**Comment by Wssduer** [comment]
Shōu dào nín de huíxìn wǒ bù shēng gǎnjī. Qídài nín de gōngzuò shùnlì wánchéng!(Bùyòng huíxìn)








volume_up




29&nbsp;/&nbsp;5,000







翻译结果
I would be very grateful to receive your reply. Looking forward to your successful completion of the work! (no need to reply)





------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "NucleicNet/NucleicNet"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年5月30日(星期一) 凌晨2:35
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [NucleicNet/NucleicNet] Ask questions about Nucleic Net and request the datasets of it (Issue #2)





 
Hi @Wssduer,
 
Thank you for your interest! We are preparing to release the code to build the dateset as well as training the model in the coming summer. We are still finalizing the format as the dataset is pretty huge in size (&gt;280GB.) and adjusting to recent changes in PDB (api and obsolete entries). Will surely let you know when we finished.
 
Best,
 Jordy

**Comment by jhmlam** [comment]
@Wssduer We are uploading files! Don't press download yet~ Will probably deliver within a few days. Stay tuned!

**Comment by jhmlam** [comment]
Release checkpoints TODO
- [x] Check if Halo Coordinate in .pkl and .xyz are same. (float32 vs 3rd Decimal) If yes, just read the xyz. Provide a function to read xyz e.g. csv dataframe
- [x] Separate the folds as folders so that developers can test on those folds with their own program. i.e. separate the tars into folds.
- [x] Provide a notebook that reads the folds separated into folders.
- [ ] Send Mmseq for a thorough test on learning rate. Seems to accept higher Learning rate(?)

**Comment by jhmlam** [comment]
@Wssduer 
Sorry for the giant delay! Now we have uploaded the indexing on datapoints at the protein surface. Please follow the instruction in Notebook97 for how to benchmark external program (we provided an example with sklearn's SVM and a X-to-Y dataframe mapping halo index to label e.g. AUCG). The instruction to decompress the dataframe is in Notebook00. Let me know if there is a question and welcome for trial.

Best, 
Jordy
